_includes/2017_data/team_members.html

```diff
@@ -3,7 +3,7 @@
         <div class="container">
             <div class="row">
                 <div class="col-lg-12 text-center">
-                    <h2 class="section-heading">Team Members</h2>
+                    <h2 class="section-heading">Planning Committee</h2>
```

Comment:
I think I used organizing committee in CoC, we should make sure to use always the same "title"